### DISCUSSION TITLE
Multiple Domains / Websites to Multiple Repositories under one GitHub account ?

### DISCUSSION THREAD

**Discussion by anjanesh** [community/community#35433] [Other Feature Feedback, Questions, & Ideas]
Is it possible to deploy multiple domains to multiple repositories under one GitHub user account ? Or is it reseticted to one repository only ?
```
domain1.com -> github.com/username/repo1's main branch
domain2.com -> github.com/username/repo2's main branch
domain3.com -> github.com/username/repo3's main branch
```

**Comment by yoannchaudet** [comment]
Each Pages site can have its own custom domain. So yes this is possible.

**Comment by anjanesh** [comment]
Thanks for the clarification - is there any violation if I host, say, 50 of my friend's / client's static websites on their respective (50) repositories under my GitHub account ?

I am ready to upgrade to the GitHub PRO for this requirement if needed.

**Reply by yoannchaudet** [reply]
You can read GitHub Pages' terms of service right here: https://docs.github.com/en/site-policy/github-terms/github-terms-for-additional-products-and-features#pages.
